libnmstate/nm/applier.py

```diff
@@ -87,6 +85,8 @@ def apply_changes(context, net_state, save_to_disk):
         _create_proxy_ifaces_desired_state(ifaces_desired_state)
     )
 
+    pending_con_profiles = []
```

Comment:
I am fine with all the changes. I will run the integration tests several time so see if there are random errors. Anyway, could you add a comment here? I assume for the context that `pending_con_profiles` is a list of tuples, the first element of the tuple is the current_profile for that device and the second element is the new_profile. Adding a little comment with this information will be nice. **NOT A MUST**, thank you!

Reply:
Comment included. Thanks.